src/MicrosoftTeams.ts

```diff
@@ -2165,6 +2165,24 @@ export namespace tasks {
     callbacks[messageId] = submitHandler;
   }
 
+  /**
+   * Update height/width task info properties.
+   * @param taskInfo An object containing width and height properties
+   * @param appIds Helps to validate that the call originates from the same appId as the one that invoked the task module
+   */
+  export function updateTask(
+    taskInfo?: object,
+    appIds?: string | string[]
```

Comment:
I don't think the appIds are necessary here. They are only needed in submitTask to secure the channel we use to send data to the app's bot. In this case there is no data flowing back to the bot and the origin of these calls is already secured via the validDomains.

Reply:
Gotcha, that's good to know. I'll adjust the PR accordingly.